src/installing-guacamole.md

```diff
@@ -435,6 +435,7 @@ guacamole-server version 1.5.5
       guacenc .... yes
       guaclog .... yes
 
+   FreeRDP plugins: /usr/lib/pkgconfig/../../lib/freerdp3
```

Comment:
It's a little weird to show freerdp2 in the output above with freerdp3 paths here, no?

Reply:
Oops, yes. I'll make this consistent.

Reply:
Fixed via rebase.